woocommerce-gateway-stripe.php

```diff
@@ -645,21 +646,24 @@ protected function enable_upe( $settings ) {
 							'yes'
 						);
 						// ENABLE UPE METHOD
-						$settings['upe_checkout_experience_accepted_payments'][] = $method_class::STRIPE_ID;
+						$payment_methods_to_enable[] = $method_class::STRIPE_ID;
 					}
 
 					if ( 'stripe' === $lpm_gateway_id && isset( $this->stripe_gateway ) && $this->stripe_gateway->is_enabled() ) {
```

Comment:
This might be a good opportunity to use the constants here (and below).

Reply:
Thank you, Wesley! I'll update this in another PR.